server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+const pokemonController = require('./controllers/pokemon');
+
+//middleware
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(methodOverride('_method'));
+app.use('/pokemon', pokemonController);
+app.use(express.static('public'));
+
+app.get('/',(req, res) => {
+	res.send('This is the server responding to a get request')
+})
+
+app.listen(3000, () => {
+	console.log('server is listening on port 3000');
+})
```

Comment:
i'd recommend organizing this file a little more carefully.  have a separate section for controllers and put all your controller stuff there (even though they are implemented the same way middleware is).  example:
```
const express = require('express');
const app = express();
const bodyParser = require('body-parser');
const methodOverride = require('method-override');

//middleware
app.use(bodyParser.urlencoded({ extended: false }));
app.use(bodyParser.json());
app.use(methodOverride('_method'));
app.use(express.static('public'));

// controllers
app.use('/pokemon', pokemonController);
const pokemonController = require('./controllers/pokemon');

app.get('/',(req, res) => {
	res.send('This is the server responding to a get request')
})

app.listen(3000, () => {
	console.log('server is listening on port 3000');
})
```